src/render/references.hbs

```diff
@@ -15,5 +16,7 @@
 {{/if}}
 </details>
 </div>
-<div class="hide">Citation key: {{citation_key}}</div>
-{{/if}}
\ No newline at end of file
+
+{{/if}}
+
+<br/>
```

Comment:
Here, a newline at the end of the file it missing.

Reply:
Done